robot/Cumulus/tests/browser/crlps/total_gifts_current_year.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/CustomizableRollupsPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
```

Comment:
Are you using NPSPSettings or ObjectManager pages here ? If not, can you please remove them.

Reply:
Will remove